src/intl/bg/page-developers-tutorials.json

```diff
@@ -15,7 +15,7 @@
   "page-tutorial-submit-btn": "Публикувайте урок",
   "page-tutorial-submit-tutorial": "За да публикувате урок, ще трябва да използвате GitHub. Добре дошли сте да зададете въпрос или заявка за сливане.",
   "page-tutorial-subtitle": "Добре дошли в нашия подбран списък с уроци за общността.",
-  "page-tutorial-tags-error": "Никои от уроците нямат всички тези тагове",
+  "page-tutorial-tags-error": "Никои от уроците нямат всички тези тагове <b>все още</b>",
```

Comment:
No biggie but for a11y I think `<strong>` is more semantic than `<b>` in this usage